src/app/modules/ix-table2/interfaces/table-column.interface.ts

```diff
@@ -6,7 +6,7 @@ export abstract class ColumnComponent<T> {
   sortBy?: (row: T) => string | number;
   sortable?: boolean;
 
-  protected row: T;
+  row: T;
```

Comment:
Keep in mind that fix in master is better and should be used instead of this one.

Reply:
> 

yeah, sorry, didn't update it - because I started my master fix here and continued as separate branch